package.json

```
@@ -43,7 +43,7 @@
     "@bugsnag/plugin-react": "^7.17.0",
     "@floating-ui/react": "^0.19.2",
     "@opentripplanner/base-map": "^3.0.16",
-    "@opentripplanner/core-utils": "^11.2.3",
+    "@opentripplanner/core-utils": "11.3.0",
```

Any reason to pin the core-utils version?
```suggestion
   "@opentripplanner/core-utils": "^11.3.0",
```